_dashboards/quickstart.md

```diff
@@ -1,60 +1,73 @@
 ---
 layout: default
-title: Quickstart guide
+title: Dashboards quickstart
```

Comment:
do we want to say OpenSearch Dashboards quickstart? Since dashboards is a plugin within OpenSearch Dashboards?